projects/kubernetes-sigs/image-builder/build/build_raw_image.sh

```diff
@@ -81,9 +81,15 @@ MAX_RETRIES=15
 for i in $(seq 1 $MAX_RETRIES); do
     echo "Attempt $(($i))"
 
+    AZ_LIST=($(aws ec2 describe-availability-zones --query "AvailabilityZones[*].ZoneName" --output text))
```

Comment:
didnt we have this problem before somewhere else?

Reply:
i think kops tests?

Reply:
hmm no, im thinking maybe in the e2e tests framework? idk tho, but i feel like weve had this discussion before....

Reply:
I think that was throttling based on requests to the RunInstances API call, which we fixed by implementing retries. I feel like this is the first time I've seen the insufficient capacity one, but I may be mistaken.